index.js

```diff
@@ -286,6 +288,12 @@ export function execaSync(rawFile, rawArgs, rawOptions) {
 	};
 }
 
+const validateSyncOptions = ({ipc}) => {
+	if (ipc) {
+		throw new TypeError('The "ipc: true" option cannot be used with synchronous methods.');
```

Comment:
`child_process.spawnSync()` does not allow IPC. It would not make sense anyway, since `childProcess.send()` would not be available.